_topic_maps/_topic_map.yml

```diff
@@ -2349,10 +2349,15 @@ Topics:
 - Name: Application backup and restore
```

Comment:
Your indentation is off, and it looks like you have some extra lines. This section should look like this:

```
- Name: Application backup and restore
  Dir: application_backup_and_restore
  Topics:
  - Name: OADP features and plugins
    File: oadp-features-plugins
  - Name: Issues with Velero and admission webhooks
    File: migration-debugging-velero-admission-webhooks
  - Name: Installing and configuring OADP
    Dir: installing
    Topics:
    - Name: About installing OADP
      File: about-installing-oadp
    - Name: About installing OADP
      File: about-installing-oadp
    - Name: Installing and configuring OADP with AWS
      File: installing-oadp-aws
    - Name: Installing and configuring OADP with Azure
      File: installing-oadp-azure
    - Name: Installing and configuring OADP with GCP
      File: installing-oadp-gcp
    - Name: Installing and configuring OADP with MCG
      File: installing-oadp-mcg
    - Name: Installing and configuring OADP with ODF
      File: installing-oadp-ocs
    - Name: Uninstalling OADP
      File: uninstalling-oadp
  - Name: Backing up and restoring
    Dir: backing_up_and_restoring
    Topics:
    - Name: Backing up applications
      File: backing-up-applications
    - Name: Restoring applications
      File: restoring-applications
  - Name: Troubleshooting
    File: troubleshooting
- Name: Control plane backup and restore
  Dir: control_plane_backup_and_restore
```